ci/pipelines/concourse.yml

```diff
@@ -782,20 +798,174 @@ jobs:
       vars_files:
       - prod/wings/vars.yml
 
-- name: push-docs
+- name: shipit
   public: true
+  serial_groups: [version]
+  plan:
+  - get: concourse
+    passed:
+    - build-rc
+    - k8s-topgun
```

Comment:
neat! I like the fact that we end up having the `concourse-rc` image tested running in a containerized env before promoting it 

<img width="2684" alt="screen shot 2019-03-01 at 11 13 04 am" src="https://user-images.githubusercontent.com/3574444/53651149-07d62d00-3c14-11e9-8165-b9e0b238ec1a.png">